core/src/main/java/io/grpc/ClientStreamTracer.java

```diff
@@ -46,6 +46,12 @@ public void headersSent() {
   }
```

Comment:
Renamed in the PR, because since the tracer API has not been released, it would be easier for me to back-port the rename to the 1.3.x branch right now than doing it after the release.